src/components/DataRecordingGridItem/styles.module.css

```
@@ -0,0 +1,21 @@
+.flash-animation {
+  -webkit-animation: flash 2s;
+  animation: flash 2s;
+}
+@-webkit-keyframes flash {
+  0%,
+  10% {
+    background-color: #4040ff44;
+  }
+  100% {
+  }
+}
```

The prefixing should happen my magic where necessary.

```suggestion
.flash-animation {
  animation: flash 2s;
}
```

---

👍  [82643d2](https://github.com/microbit-foundation/ml-trainer/pull/420/commits/82643d2f528e8b9308bc02786f02e989fa1d14e4)